### DISCUSSION TITLE
Please guide me how to run this project :(

### DISCUSSION THREAD

**Discussion by miroshar-success** [lobehub/lobehub#1905] [Development Question] [answered]
I am new in this project.
Please write here if there is somebody to know how to run this project.
Thanks

**Comment by arvinxx** [comment, marked as answer]
You can checkout the document: https://chat-docs.lobehub.com/en/self-hosting/start

**Comment by miroshar-success** [comment]
Thanks for you reply,
Is it possible to run on my local environment?

**Reply by arvinxx** [reply]
Yes

**Comment by miroshar-success** [comment]
Oh. That sounds good.
Could you guide me? :)

**Reply by arvinxx** [reply]
Check this https://chat-docs.lobehub.com/en/self-hosting/platform/docker

**Comment by miroshar-success** [comment]
Thanks. I run project on local environment.
I want to change this logo.
![Lobe 1](https://github.com/lobehub/lobe-chat/assets/157709103/9dfa414c-9d8e-4fd1-b9c7-906474c55bab)
Can you see this image?
I want to customize these items as mine.
For example, I am going to change 2 to mine. I tried to change, but it fails. :)
Could you guide me?
Thanks for your support.

**Reply by arvinxx** [reply]
We don't have a guide of customization for free. If you want a custom version for business, please contact hello@lobehub.com for a paid commercial license.

**Comment by miroshar-success** [comment]
OK. Thanks. Got it. 
Then Could you tell me how to build docker image for this project?
This project already include docker file. So I tried to build docker image using that docker file, but it appears error below.

28: RUN pnpm run build:docker # run build standalone for docker version

What's wrong? What can I do for this?

**Comment by miroshar-success** [comment]
Did you try to build docker image in Windows?
I get error in install npm. 
I heard docker works well in mac or ubuntu. not windows.
Then npm install is possible in windows?
I am not sure.
Could  you guide me?
